### PR TITLE
BUGFIX: Use FlashMessageService

### PR DESCRIPTION
**Pull request by kdambekalns**

### CODE REVIEW

File: Classes/Core/Runtime/FormRuntime.php [hunk 1]
```diff
@@ -191,7 +182,7 @@ protected function initializeFormStateFromRequest()
             $this->formState = new FormState();
         } else {
             $serializedFormState = $this->hashService->validateAndStripHmac($serializedFormStateWithHmac);
-            $this->formState = unserialize(base64_decode($serializedFormState));
+            $this->formState = unserialize(base64_decode($serializedFormState), [FormState::class]);
```
Comment (daniellienert) [inline Classes/Core/Runtime/FormRuntime.php]:
```suggestion
            $this->formState = unserialize(base64_decode($serializedFormState), ['allowed_classes' => [FormState::class]]);
```

Comment (kdambekalns) [reply Classes/Core/Runtime/FormRuntime.php]: Thanks! I knew it felt strange, but couldn't pinpoint that feeling…

**Comment by bwaidelich** [comment]
@kdambekalns @daniellienert it's been a while, but can you recall how the `allowed_classes` change is related to this? It seems to break some things

**Comment by kdambekalns** [comment]
@bwaidelich Not needed to use the FlashMessageService, no. I added it to get rid of a security warning and was sure there was only ever going to be a `FormState` in need of being deserialized.

**Comment by bwaidelich** [comment]
@kdambekalns Thanks for the update, I'll take care of a bugfix

**Pull request merged**